pages/guides/developing/dstorage-playback.en-US.mdx

```diff
@@ -12,6 +12,18 @@ Playing back transcoded video from IPFS or Arweave is easy with livepeer.js! The
 
 IPFS is a network of nodes that allow you to read content from the network using a content identifier unique to the data you’re requesting, ensuring you are able to securely and verifiably get exactly what you are asking for, regardless of where the data is physically stored. Arweave is a set of nodes that are incentivized to store data permanently; content stored on the network is pulled through an Arweave gateway.
 
+<Callout type="warning" emoji="⚠️">
+  **This example illustrates how Livepeer can be used as a video processing and
+  caching layer on top of content available on IPFS or the Arweave gateway for
+  optimized video playback.** We recommend uploading via `useCreateAsset` or the API before displaying it in
+  the player for best performance. For comprehensive IPFS / Arweave support we offer a fallback option to play directly from
+  IPFS/Arweave while the video is being uploaded to Livepeer, but do not recommend it for typical usage.
```

Comment:
1. I think we lack an explanation why we don't recommend it and what happens if you do it. Will it work? Will it be slow?
2. As a side node, these messages sound very strong which makes me think if we should support these "not recommended" features at all.